javascript/javascript1/week2/formalname.js

```diff
@@ -1,14 +1,14 @@
 let fullname1, fullname2;
 
-function getFullname(firstname, surname, useFormalName = true) {
+function getFullname(firstname, surname, useFormalName) {
   if (useFormalName == true) {
     return "lord " + firstname + surname;
   } else {
     return firstname + surname;
   }
 }
 
-fullname1 = getFullname("Kiran ", "Serolla");
-fullname2 = getFullname("Kanishk ", "Kalvala");
+fullname1 = getFullname("Kiran ", "Serolla", true);
+fullname2 = getFullname("Kanishk ", "Kalvala", false);
```

Comment:
after you update the function parameters and keep the default value as `false` like this
```js
function getFullname(firstname, surname, useFormalName) {...}
```
then you can write fullname2 like this
```js
fullname2 = getFullname("Kanishk ", "Kalvala");
```
I mean without the third argument. So when you want it to be `true` you write it explicitly, otherwise no need to write it